fullcase.js

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+
+const SpecialCasing = ""+fs.readFileSync("SpecialCasing.txt");
+const CaseFolding = ""+fs.readFileSync("CaseFolding.txt");
+
+const special =
+  /^([0-9A-F]+);\s+([0-9A-F]+(?:\s+[0-9A-F]+)*)?;\s+([0-9A-F]+(?:\s+[0-9A-F]+)*)?;\s+([0-9A-F]+(?:\s+[0-9A-F]+)*)?;\s+(#.+?)?$/mg;
+
+const folding =
+  /^([0-9A-F]+);\s+([CFS])?;\s+([0-9A-F]+(?:\s+[0-9A-F]+)*)?;\s+(#.+?)?$/mg;
+
+const scData = {};
+const cfData = {};
+
+const rhex = a => typeof a === 'string' ? +('0x'+a) : +a;
+const sortHex = (a,b) => rhex(a) - rhex(b);
+
+for (const spec of SpecialCasing.matchAll(special)) {
```

Comment:
`String.prototype.matchAll` is a very recent feature. What kind of engine support are we targeting with this project? We can accomplish the same thing without it if we need.

Reply:
This is just for generating code, it's not part of the runtime of the library. So I assume that developers can find a recent enough node.